test/studies/comd/llnl/LICENSE

```diff
@@ -0,0 +1,70 @@
+/******************************************************************************
+
+CoMD-Chapel v1.0
+
+Copyright (c) 2016, Lawrence Livermore National Security, LLC.
```

Comment:
Perhaps, but I think that's on LLNL. I just copied this license from their repo.